dotnet/samples/GettingStartedWithProcesses/Step03/Step03a_FoodPreparation.cs

```diff
@@ -22,6 +22,8 @@ public class Step03a_FoodPreparation(ITestOutputHelper output) : BaseTest(output
     public async Task UsePrepareFriedFishProcessAsync()
     {
         var process = FriedFishProcess.CreateProcess();
+        process.LinkEventSubscribersFromType<FriedFishEventSubscribers>();
```

Comment:
remove FishProcess changes, instead add another step to showcase only eventSubscribers